docs/products/postgresql/concepts/pg-audit-logging.md

```diff
@@ -0,0 +1,192 @@
+  environment
+
+Compliance
+
+- Use audit logs as a regulatory compliance evidence to demonstrate that the organization
+  meets industry or state regulations during audits
+- Track access to sensitive data to comply with data privacy regulations
+
+Accountability
+
+- Have specific actions attributed to individual users to hold them accountable for their
+  activities within the system
+- Track changes to databases and systems to hold users accountable for alterations or
+  configurations
+
+Operational security
+
+- Monitor and analyze audit logs to proactively identify and resolve security incidents
+- Analyze audit logs to detect and respond to potential security threats
+
+Incident management and root cause analysis
+
+- Investigate an incident with audit logs as a detailed trail of events leading up to the
+  incidents
+- Analyze the root cause of an incident with audit logs providing data on actions and
+  events that may have led to the incident
+
+System performance optimization
+
+- Monitor and analyze system performance to identify bottlenecks.
+- Analyzing audit logs to assess resource utilization patterns and optimize the system
+  configuration
+
+Data recovery and disaster planning
+
+- Use audit logs for data restoration in case of data loss or system failure
+- Analyze audit logs to improve system resilience and disaster planning strategies by
+  identify potential points of failure
+
+Change management and version control
+
+- Use audit logs to keep a record of changes made to databases, software, and
+  configurations, ensuring a proper version control
+
+## Use cases
+
+The audit logging feature has application in the following industries:
+
+- Finance and banking
+
+  Ensuring compliance with regulatory requirements, tracking financial transactions, and
+  detecting fraudulent activities
+
+- Healthcare
+
+  Maintaining the confidentiality and integrity of patient records as well as complying
+  with privacy regulations
+
+- Government and public sector
+
+  Tracking changes in critical systems, secure sensitive data, and meet legal and
+  regulatory requirements
+
+- Information technology (IT) and software companies
+
+  Monitoring access to the systems, tracking software changes, and identifying potential
+  security breaches
+
+- Retail and e-commerce
+
+  Tracking customer data, transactions, and inventory management to ensure data integrity
+  and prevent unauthorized access
+
+- Manufacturing
+
+  Tracking changes to production processes, monitoring equipment performance, and
+  maintaining data integrity for quality control
+
+- Education
+
+  Protecting sensitive student data, tracking changes to academic records, and monitoring
+  system access for security purposes
+
+## Who is eligible
+
+Aiven for PostgreSQL® audit logging requires the following:
+
+- `[Pro Platform](/docs/platform/concepts/pro-platform)`
+- Aiven for PostgreSQL 11+ Pro Plan
```

Comment:
I think the "Pro Plan" part is redundant here as it might give the user the wrong impression that Pro Plans are coupled with PostgreSQL version.

Aiven for PostgreSQL(R) version 11 or greater.